metastats-domain/src/main/groovy/life/qbic/metastats/MetaStatsLogManager.groovy

```diff
@@ -0,0 +1,36 @@
+package life.qbic.metastats
+
+/**
+ * <class short description - One Line!>
+ *
+ * <More detailed description - When to use, what it solves, etc.>
+ *
+ * @since: <versiontag>
```

Comment:
Insert Version Tag here!
```suggestion
 * @since: <versiontag>
```

Reply:
To keep the formatting (* is missing) I added your suggestions manually: a9fd7aa